css/index.css

```diff
@@ -197,6 +198,66 @@ p {
 	background: dodgerblue;
 }
 
+/*----- About Page start -----*/
+
+.about-section{
+		margin: 20px 0 20px 0;
```

Comment:
You could use the shorthand for margin here. It'd look like `margin: 20px 0`. If you only specify two values, it will be (TB) (LR) instead of T R B L.